pkg/gitlab/jobs.go

```diff
@@ -137,16 +133,21 @@ func (c *Client) ListPipelineChildJobs(projectName string, parentPipelineID int)
 		}
 
 		for _, foundBridge := range foundBridges {
+			// Trigger job was created but not yet executed
+			// so downstream pipeline is not yet scheduled to start.
+			// Therefore no pipeline is available and bridge could be skipped.
+			if foundBridge.DownstreamPipeline == nil {
+				continue
+			}
+
```

Comment:
Im not sure whether if I skip the child pipelines here, would it be retry later? 

Or the parent pipeline ID is recorded somewhere and this would mark the parent pipeline as successfully scraped, and no retry. (If so I should return a special error here instead?)

Reply:
yes, it will be retried on the next pipeline metrics pull 👍